### PR TITLE
13054 Setup save button for FEIS

### PR DESCRIPTION
**Pull request by CeciliaCY**
### Summary
13054 Setup save button for FEIS

#### Task/Bug Number
Fix AB [13054](https://dcp-paperless.visualstudio.com/ZAP%20Portals/_workitems/edit/13054)

### Technical Explanation
Same as Draft EAS Save Button

### CODE REVIEW

File: client/app/components/packages/feis/edit.hbs [hunk 1]
```diff
@@ -1,26 +1,43 @@
...
 
-      <section class="form-section">
-        <h1 class="header-large">
-          Final Environmental Impact Statement (FEIS)
-          <small class="text-weight-normal">
-            {{if @package.dcpPackageversion (concat '(V' @package.dcpPackageversion ')')}}
-          </small>
-        </h1>
+  <div class="grid-x grid-margin-x">
+      <div class="cell large-8">
```
Comment (allthesignals) [inline client/app/components/packages/feis/edit.hbs:8]: I know this isn't part of your commit, but it looks like the spacing is messed up here again. 

It shouldn't be 4 spaces, it should only be 2 spaces for tabbing. 

Sorry we don't have a better linter to help with this. Can you fix please? Then, you'll need to re-align the tabbing for the rest of the diff below.

File: client/app/models/project.js [hunk 2]
```diff
@@ -112,8 +112,8 @@ export default class ProjectModel extends Model {
 
   get eisPackages() {
     const eisPackages = [
-      ...this.deisPackages,
       ...this.feisPackages,
+      ...this.deisPackages,
```
Comment (godfreyyeung) [inline client/app/models/project.js:116]: Weird, this change should already be in develop

File: client/app/components/packages/feis/edit.hbs (hunk 1 above)
Comment (allthesignals) [inline client/app/components/packages/feis/edit.hbs:8]: I know this isn't part of your commit, but it looks like the spacing is messed up here again.

It shouldn't be 4 spaces, it should only be 2 spaces for tabbing.

Sorry we don't have a better linter to help with this. Can you fix please? Then, you'll need to re-align the tabbing for the rest of the diff below.

**Review by allthesignals — COMMENTED** [review]
This looks good but where is the acceptance test for FEIS?

File: client/app/models/project.js (hunk 2 above)
Comment (godfreyyeung) [reply client/app/models/project.js:116]: Why is this diff still rendering? Very strange...

**Review by godfreyyeung — APPROVED** [review]
After the fixes Matt mentioned, this should be good to go.
Also, see if rebasing this branch off develop fixes the strange behavior I mentioned for the `eisPackages` getter.

File: client/app/models/project.js (hunk 2 above)
Comment (CeciliaCY) [reply client/app/models/project.js]: Please check again

File: client/app/components/packages/feis/edit.hbs (hunk 1 above)
Comment (CeciliaCY) [reply client/app/components/packages/feis/edit.hbs]: @allthesignals I think this is related to my vscode, I just updated it tab = 2 spaces. Please check again. Thanks!

**Pull request merged**